src/domain/operation_mode.py

```diff
@@ -26,10 +28,13 @@ class OperationModeTypes(Enum):
 
     # Signals
     update_image = Signal(str)
+    update_actuator_status = Signal()
     update_info = Signal()
     run_finished = Signal()
     run_progress = Signal(int)
 
+    flag_stop_update_actuators_thread = False
```

Comment:
Why this is a class attribute? actuators threads are binded to operation mode instantiation as every thread must be created and destroyed following operation mode derived classes lifecycle.

this flag should be moved to object attributes pool.

Reply:
Maybe it's wanted / expected. This value is referenced and also set using self.flag_stop_update_actuators_thread in the code, so that every instance will have its own value.